src/igvfd/schemas/changelogs/construct_library_set.md

```diff
@@ -7,6 +7,8 @@
 ### Schema version 7
 
 * Require `release_timestamp` for any objects with `released`, `archived`, or `revoked` status.
+* Add `orf_list`.
```

Comment:
I think these entries were added to the wrong section. It is probably a new header "Minor changes since schema version 8"

Reply:
done